samples/FormatFilterSample.Web/Startup.cs

```diff
@@ -41,7 +38,7 @@ public void Configure(IApplicationBuilder app)
                 routes.MapRoute("optionalroute",
                                 "{controller}/{action}.{format?}",
                                 new { controller = "Home", action = "Index" });
```

Comment:
Suspect the original code was centred around the "It Just Doesn't Matter" mantra i.e. make few changes from something else. But, sure, I can make the change.